paddle/fluid/operators/detection/roi_perspective_transform_op.cu

```diff
@@ -35,12 +32,12 @@ namespace operators {
 
 template <typename T>
 __device__ bool GT_E(T a, T b) {
-  return (a > b) || fabs(a - b) < 1e-4;
+  return (a > b) || fabsf(static_cast<float>(a - b)) < 1e-4;
 }
 
 template <typename T>
 __device__ bool LT_E(T a, T b) {
-  return (a < b) || fabs(a - b) < 1e-4;
+  return (a < b) || fabsf(static_cast<float>(a - b)) < 1e-4;
```

Comment:
fabsf only works for float type. For float16, it will goes wrong. These function should be put into 
float16_helper.h, and create template specification for different data type.

Reply:
ok